src/Explorer/Explorer.tsx

```diff
@@ -1568,160 +1548,115 @@ export default class Explorer {
       return false;
     });
   }
-
   public openDeleteCollectionConfirmationPane(): void {
```

Comment:
@hardiknai-techm can you also remove all these methods from Explorer? For example instead of having `openDeleteCollectionConfirmationPane` just call `useSidePanel.getState().openSidePanel()` from the place that is calling this. It must already have access to Explorer since it is calling Explorer.openDeleteCollectionConfirmationPane